Modules/Console/Source/init.luau

```diff
@@ -705,12 +705,20 @@ function Console.Interface.is(object: Console?): boolean
 	return metatable and metatable.__type == Console.Type
 end
 
-export type Console = typeof(Console.Prototype) & {
-	id: string,
-	level: number,
-	schema: string,
-	enabled: boolean,
-	logs: {},
-}
+export type Console = typeof(setmetatable(
```

Comment:
Resolved in an alternative media platform - the question being, do we want to adhere to a standard or do we want to adhere to simplicity?

Going to go with standard for now, but this may change in the future when I look into making all of the modules i've developed strictly typed.